### ISSUE TITLE
[BUG] Breadcrumb - irregular spacing

### ISSUE THREAD

**Issue by ChristineLamontagne** [wet-boew/wet-boew#9042]
**Describe the bug**
This is small but it got everyone in my team pointing it out.  The spacing between elements is off.  While the  spacing between the > and a breadcrumb with a link seems to be 2 or 3 characters, the space between the > and the last element (which is not a link) is only 1 character.

**To Reproduce**
1. Go to https://noc.esdc.gc.ca/Home/Welcome/2208bd8a38c74d238bc83bc58406f7dd
2. To better see the issue click on 'About the NOC'
3. Look at the spacing between the breadcrumbs and the >

**Expected behavior**
I would expect the same space 1 character space between each entries and the >

**Comment by duboisp** [comment]
@ChristineLamontagne the breadcrumb supported design pattern are containing items that are linked text.

I think that DTO is currently reviewing the guidance around the breadcrumb to avoid page title repetition. I suggest you to not repeat the page title (h1) as the last breadcrumb item as demoed in the link you provided.

@delisma do you have anything to add?

**Comment by ChristineLamontagne** [comment]
@duboisp  the problem is that there is padding for the list items and there is padding inside the list items for the links which creates monster spacing between each > and unbalance it when you have normal text.  Removing the extra padding for the links would be more elegant in my humble opinion.

**Comment by delisma** [comment]
@ChristineLamontagne You're right the spacing after the angle bracket (`>`) is off, and it's supposed to be the same on both side. But as @duboisp mentioned, the current page isn't supposed to be listed in the trail, as per the [guidance on breadcrumb trail](https://design.canada.ca/common-design-patterns/breadcrumb-trail.html). It seems that the spacing issue occurs on non-links trails, which in your case, happens on the trail of the current page. [GCWeb](https://github.com/wet-boew/gcweb) wasn't design to support plain text breadcrumbs for the current page.
